docs/advanced-topics/hadoop.md

```diff
@@ -72,68 +72,37 @@ by leveraging Apache Spark.
 
 ### OLAP Traversals with Spark Local
 
-The backend demonstrated here is Cassandra for the OLAP example below.
-Additional configuration will be needed that is specific to that storage
-backend. The configuration is specified by the
+OLAP Example below are showing configuration examples for default support 
```

Comment:
I don't know how correctly to construct this sentence but I think we should use `supported` word here instead of `support`.